src/api/apiFetch.js

```diff
@@ -37,12 +37,13 @@ export const apiFetch = async (
   params: $Diff<$Exact<RequestOptions>, {| headers: mixed |}>,
 ) => fetch(new URL(`/${apiVersion}/${route}`, auth.realm).toString(), getFetchParams(auth, params));
 
+/** (Caller beware! Return type is the magic `empty`.) */
```

Comment:
> `fetch` having a type of `any` in the libdef RN provides

Yeah; it'll be nice to get facebook/react-native@6651b7c59, looks like in RN v0.65.

Reply:
Ah indeed, neat!